uwblueprint/ccbc
# [Bug] Fix Series Title Search

## Notion ticket link

[Search doesn’t work on series title](https://www.notion.so/uwblueprintexecs/Search-doesn-t-work-on-series-title-e6a60634fde44d568f9d45c7ab1e1bcb)



## Implementation description
* Fixed Migration Scripts that errored due to not existing columns / constraints
* Used a try-catch block for constraint. ~ Prob not a good style. 



## Steps to test
1. Can search by title now in admin table
2. Check running the migrations from Github ReadMe



## What should reviewers focus on?
* Code style and quality


## Checklist
